TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -320,7 +321,65 @@ class RouteOptionsViewController: UIViewController {
         navigationItem.hidesBackButton = true
         searchBarView.searchController?.isActive = true
     }
-
+    
+    @objc func rerenderLiveTracking(sender: Timer) {
+        // Reload table every time update timer is fired
+        routeResults.reloadData()
+    }
+    
+    private func getDelay(tripId: String, stopId: String) -> Future<Response<Int?>> {
+        return networking(Endpoint.getDelay(tripID: tripId, stopID: stopId)).decode()
+    }
+
+    @objc func updateLiveTracking(sender: Timer) {
+        for routesArray in routes {
+            for (index, route) in routesArray.enumerated() {
```

Comment:
just a little confused why we have two nested loops here –– is `routesArray` corresponding to a single route, and `(index, route)` covers the case of transfers? this might be useful to have in a comment, just to make it clearer for anyone in the future!